features/main/content_negotiation.feature

```diff
@@ -110,8 +110,8 @@ Feature: Content Negotiation support
     Then the response status code should be 406
     And the header "Content-Type" should be equal to "application/problem+json; charset=utf-8"
 
-  Scenario: If the request format is HTML, the error should be in HTML
```

Comment:
Can you keep this text (and duplicate it for XML if appropriate)? It checks that the documentation for any URL feature works (for instance when you open `http://localhost/books/1`, Swagger UI shows up and automatically runs the AJAX request).